index.html

```diff
@@ -41,19 +40,158 @@ <h1 id="idh1">
+            <ul class="skills">
+                <li>
+                    <a href="#"><button class="b1">html</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b2">css</button></a>
+                </li>
+                <li>
+                    <a href="#"><button class="b3">javascript</button></a>
+                </li>
+            </ul>
+            <a href="#"><button class="b4">See Project</button></a>
+        </section>
+        <section class="about-myself">
+            <h1>About <br> Myself</h1>
+            <p id="about-myself">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+            </p>
+            <div id="socialid">
+                <a href="#" id="connectid">
+                    LET'S CONNECT
+                </a>
+                <ul class="ic">
+                    <li><a href="https://github.com/CarlosIgreda" target="_blank" rel="noopener"><img
+                                src="./images/Github.svg" alt="Github"></a></li>
+                    <li><a href="https://www.linkedin.com/in/carlos-jes%C3%BAs-igreda-rivas-323763116/" target="_blank"
+                            rel="noopener"><img src="./images/Linkedin.svg" alt="Linkedin"></a></li>
+                    <li><a href="https://www.angellist.com/" target="_blank" rel="noopener"><img
+                                src="./images/AngelList.svg" alt="AngelList"></a></li>
+                    <li><a href="https://twitter.com/carlosigreda" target="_blank" rel="noopener"><img
+                                src="./images/Twiter.svg" alt="Twiter"></a></li>
+                    <li><a href="https://medium.com/" target="_blank" rel="noopener"><img src="./images/Medium.svg"
+                                alt="Medium"></a></li>
+                </ul>
+            </div>
+            <a href="#"><button class="b5">Get my resume</button></a>
+            <div id="idlanguage" class="more-info">
+                <h2>Languages</h2>
+                <img src="./images/down-arrow.svg" alt="down-arrow">
+            </div>
+            <ul id="ul-about">
+                <li><img src="./images/javascript-icon.svg" alt="javascript"><span>JavaScript</span></li>
+                <li><img src="./images/html-icon.svg" alt="html"><span>HTML</span></li>
+                <li><img src="./images/css-icon.svg" alt="css"><span>CSS</span></li>
+            </ul>
+            <div class="more-info separator">
+                <h2>Frameworks</h2>
+                <img src="./images/right-arrow.svg" alt="right-arrow">
+            </div>
+            <div class="more-info separator">
+                <h2>Skills</h2>
+                <img src="./images/right-arrow.svg" alt="right-arrow">
```

Comment:
- [ ] please The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`

Reply:
Done @tufoinnkuo10 . I've already made a `ul` tag nested in another `ul` tag.